sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -277,6 +279,62 @@ export type HealthcareEntityRelationType = string;
 // @public
 export type InnerErrorCodeValue = string;
 
+// @public
+export const enum KnownHealthcareEntityCategory {
+    // (undocumented)
```

Comment:
curious, what does (undocumented) means? that you still need to write a docstring for it?

Reply:
Yes. I am exporting it from the generated layer which was just following the swagger enum